modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/XMLUpgradeDTDVersionCheck.java

```diff
@@ -34,11 +34,11 @@ protected String doProcess(
 		_upgradeToVersion = getAttributeValue(
 			SourceFormatterUtil.UPGRADE_TO_VERSION, absolutePath);
 
-		if (_upgradeToVersion == null) {
+		if ((_upgradeToVersion == null) && !fileName.endsWith(".xml")) {
```

Comment:
Same thing here. It should be `_upgradeToVersion == null || !fileName.endsWith(".xml")`.